resources/ajax_htmldata/getIssuesList.php

```diff
@@ -80,7 +80,7 @@ function generateIssueHTML($issue,$associatedEntities=null) {
 
 			if(count($orgIssues) > 0) {
 				foreach ($orgIssues as $issue) {
-					echo generateIssueHTML($issue,array(array("name"=>$orgData['organization'],"id"=>$organization->organizationID,"entityType"=>1)));
+					echo generateIssueHTML($issue,array(array("name"=>$orgData['organization'],"id"=>$organization->primaryKey,"entityType"=>1)));
```

Comment:
Why this change? If organizationID doesn't produce the expected value, then there may be an underlying problem.